packages/upgrade/test/commands/hello/index.test.ts

```diff
@@ -0,0 +1,10 @@
+import {expect, test} from '@oclif/test'
```

Comment:
looks like boilerplate. should this file be committed?

Reply:
https://github.com/nomad-xyz/monorepo/pull/349/commits/cacaa595409bb5807a628524879583d22c6f7f56